sci-electronics/librepcb/metadata.xml

```diff
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <!-- maintainer-needed -->
+   <maintainer type="project" proxied="proxy">
+     <email>sci-electronics@gentoo.org</email>
+     <description>Gentoo Electronics Project</description>
+   </maintainer>
+   <maintainer type="person" proxied="yes">
+     <email>ktrace@yandex.ru</email>
+     <name>Victor V. Kustov</name>
+   </maintainer>
+   <longdescription lang="en">
+     LibrePCB is a free EDA software to develop printed circuit boards.
```

Comment:
Looks good :+1: 

We've just got one complaint from the QA bot::
```
RedundantLongDescription | metadata.xml longdescription closely matches DESCRIPTION
```
We should either just drop the long description, or replace it with a longer one.

Reply:
ready